design/standard/templates/class/datatype/view/ezimage.tpl

```diff
@@ -3,3 +3,15 @@
     <label>{'Max file size'|i18n( 'design/standard/class/datatype' )}:</label>
     <p>{$class_attribute.data_int1}&nbsp;MB</p>
 </div>
+
+<div class="block">
+    <p>
+        <b>
+        {if $class_attribute.data_int2|ne( 0 )}
```

Comment:
There's usually a reason I would do something like this, although  I can't remember why I did it here.  Also I have been consistent as you have commented on my usage later on.  It is easy for me to change it and check it.